DigitalExcellence/dex-frontend
# Feature/217 project icon

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Add a new project
2. Drag and Drop/ browse an image in the box
3. An image should be previewed underneath the drop area and the file should start uploading
4. Create the project and check if the icon has been added to the project (in the database, it won't be displayed yet).

## Link to issue
[#266](https://github.com/DigitalExcellence/dex-frontend/issues/266)
Closes: #266
